.gitignore

```diff
@@ -1,3 +1,13 @@
+
+# to avoid commiting data
+data/derived
```

Comment:
I suspect that it will actually be better to make the `data/derived` a datalad subdataset so we can version control the output data